release-1.7/release-1.7.md

```diff
@@ -1,47 +1,63 @@
 # Proposed timeline for 1.7
 We will begin the 1.7 release the first week of April 2017.
 
-The proposal below is identical in layout to the 1.5 and 1.6 plans with some
+The proposal below is similar in layout to the 1.5 and 1.6 plans with some
 modifications:
 - as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
 
 ## 1.7 Release Schedule
 - April 3 (Mon) coding start (7w)
 - April 26 (Weds) features repo freeze
-- April 6th (Thurs) v1.7.0-alpha.1
-- April 20th (Thurs) v1.7.0-alpha.1
-- May 31st (Weds) Start code freeze; Cut release-1.7 branch and v1.7.0-beta.0; Feature Burndown Meetings begin
-- June 13st (Weds) v1.7.0-beta
-- June 20st (Weds) v1.7.release-candidate
-- June 28th (Weds) - v1.7.0 (release!)
+- April 6th (Thurs): v1.7.0-alpha.1
+- April 19th (Weds): v1.7.0-alpha.2
+- May 3 (Weds): v1.7.0-alpha.3
+- May 17(Weds): v1.7.0-alpha.4
+- May 31(Weds): v1.7.0-beta.0
+- Jun 1 (Thurs): Code Freeze 
+- Jun 7 (Weds): v1.7.0-beta.1
+- June 14 (Weds): v1.7.0-beta.2
+- June 19th (Mon): v1.8.0-alpha.1
+- June 21th (Weds): v1.7.release-candidate, v1.8.0-alpha.1
```

Comment:
See above comment. It seems like v1.8.0-alpha.1 shouldn't be cut again on June 21th.